client/flutter/lib/carousel_page.dart

```diff
@@ -63,6 +68,10 @@ class CarouselView extends StatelessWidget {
                 padding: EdgeInsets.only(bottom: 20),
                 child: pageViewIndicator(context)),
           ),
+          GestureDetector(
+          onTap: ()=>pageController.page == this.items.length-1?null:pageController.nextPage(duration: Duration(milliseconds: 500), curve: Curves.easeInOutCubic)
```

Comment:
Nit: Is the spacing around operators here intentional? Should we incorporate dartfmt in our pipeline to standardize style?

Reply:
When I wrote this my IDE wasn't working, I will re push after formatting

Reply:
@sstur thx, just fixed

Reply:
@sstur Check #362.